packages/functional-tests/tsconfig.json

```diff
@@ -1,8 +1,18 @@
 {
   "extends": "../tsconfig.packages.json",
-  "include": [
-    "./src"
+  "files": [

```

Comment:
Why this change? I want to avoid the use of the "files" section. It's one more thing to maintain. Was "include": [ "./src" ] not working for you?

Reply:
This is strictly temporary. I didn't want it building files I hadn't fixed yet.